src/loadData.js

```diff
@@ -87,7 +84,7 @@ const elaborate = (data) => {
     gen_f: categoriesAndAges.reduce(sumDoseX("sesso_femminile"), 0),
   };
 
-  const timestamp = categoriesAndAges[0].data;
+  const timestamp = categoriesAndAges[0].ultimo_aggiornamento;
```

Comment:
We can use data from `https://raw.githubusercontent.com/italia/covid19-opendata-vaccini/master/dati/last-update-dataset.json` here

Reply:
wow sure